.gitlab/build_tioga.yml

```diff
@@ -0,0 +1,27 @@
+####
+# This is the shared configuration of jobs for tioga
+.on_tioga:
+  tags:
+    - shell
+    - tioga
+  rules:
+    - if: '$CI_COMMIT_BRANCH =~ /_qnone/ || $ON_TIOGA == "OFF"' #run except if ...
+      when: never
+    - when: on_success
+
+####
+# Template
+.src_build_on_tioga:
+  stage: build
+  variables:
+    ALLOC_COMMAND: "flux mini run -t10m -n1"
+  extends: [.build_script, .on_tioga]
+
+####
+# Build jobs
+tioga-clang_14_0_0_hip:
+  variables:
+    COMPILER: "clang@14.0.0_hip"
+    HOST_CONFIG: "tioga-toss_4_x86_64_ib_cray-${COMPILER}.cmake"
```

Comment:
Either this `HOST_CONFIG` variable here or the host-config file name itself needs to change to match, I think.

Reply:
That was definitely the issue, thanks.